app/views/service_providers/_form.html.erb

```diff
@@ -20,13 +20,13 @@
                  input_html: { class: 'usa-input' },
                  label_html: { class: 'usa-input-required'},
                  label: 'App name',
-                 hint: I18n.t('service_provider_form.app_name_html') %>
+                 hint: t('service_provider_form.app_name_html') %>
```

Comment:
`t` is basically just an alias to `I18n.t`, I am surprised this made a difference?

Reply:
we were also surprised. (it's also documented [here](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations) in a lil info box)

Reply:
Ruby is a language of many mysteries.